src/tilemaps/mapdata/LayerData.js

```diff
@@ -29,7 +29,7 @@ var LayerData = new Class({
         if (config === undefined) { config = {}; }
 
         /**
-         * [description]
+         * The name of the layer(specified in Tiled)
```

Comment:
Can you add a space before the open brace please. In fact, I think just `The name of the layer, if specified in Tiled.` would be fine.